entangled/commands/watch.py

```diff
@@ -25,7 +26,8 @@ def on_any_event(self, event: FileSystemEvent):
         path = Path(event.src_path)
         if path.is_relative_to(Path("./.entangled")):
             return
-        if any(path.is_relative_to(p) for p in self.watched):
+        if any(path.is_relative_to(p.absolute()) for p in self.watched):
+            os.sync()
```

Comment:
Maybe do a `os.sync()` after running entangled's `sync`, if it changes anything at all.